### ISSUE TITLE
Tooltip: Not accessible because aria-describedby is not set.

### ISSUE THREAD

**Issue by skairunner** [primefaces/primereact#8508]
### Describe the bug

The [tooltip documentation](https://primereact.org/tooltip/#accessibility) states:

> Tooltip component uses tooltip role and when it becomes visible the generated id of the tooltip is defined as the aria-describedby of the target.

This does not actually happen.

### Reproducer

https://stackblitz.com/edit/vitejs-vite-wjimyrsw?file=src%2FApp.tsx

### System Information

```Shell
PrimeReact 10.9.7, you can see it on the Primereact demo site: https://primereact.org/tooltip/#delay
```

### Steps to reproduce the behavior

1. Go to the stackblitz.
2. Use the inspector to select either button or the input.
3. Hover over the element, showing the tooltip.
4. Note that `aria-describedby` is **not** set on either the button or the input.

This happens both with the standalone and the integrated tooltip.

### Expected behavior

Expected `aria-describedby` of the target to be set.
